ironfish-cli/src/commands/service/snapshot.ts

```diff
@@ -73,7 +74,7 @@ export default class CreateSnapshot extends IronfishCommand {
   async start(): Promise<void> {
     const { flags } = await this.parse(CreateSnapshot)
 
-    const bucket = (flags.bucket || process.env.IRONFISH_SNAPSHOT_BUCKET || '').trim()
+    const bucketUrl = (flags.bucketUrl || DEFAULT_SNAPSHOT_BUCKET_URL || '').trim()
```

Comment:
Does this make it impossible to not upload to a bucket? It looks like the default would get filled in if no bucket flag is added.

Reply:
I've added a boolean `upload` flag to ensure that uploading isn't done unintentionally.